openmm_septop/python_tutorial.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>We should make it clear that we only have to make one transformation per edge unlike the RBFE protocol as it runs both the complex and solvent legs together. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>